web/src/app/schedules/calendar-subscribe/CalendarSubscribeCreateDialog.js

```diff
@@ -62,7 +62,7 @@ export default function CalendarSubscribeCreateDialog(props) {
       input: {
         scheduleID: value.scheduleID,
         name: value.name,
-        reminderMinutes: value.reminderMinutes.map(r => r && r.value),
+        reminderMinutes: [0],
```

Comment:
Let's leave a quick comment that we are defaulting to notify at time of shift on creation